src/routes/email/email.service.ts

```diff
@@ -35,7 +35,7 @@ export class EmailService {
   }): Promise<any> {
     return this.repository.verifyEmailAddress({
       ...args,
-      signer: args.account,
+      account: args.account,
     });
```

Comment:
And this can be now:
```javascript
return this.repository.verifyEmailAddress(args);
```

Reply:
[6990105](https://github.com/safe-global/safe-client-gateway/pull/944/commits/6990105ff17819e91620f80a63d99667b1dca99f)